proto/regen/ecocredit/v1/tx.proto

```diff
@@ -576,4 +599,72 @@ message MsgBridgeReceiveResponse {
   // project_id is the unique identifier of the project that was either created
   // or the existing project within which the credit batch exists.
   string project_id = 2;
-}
\ No newline at end of file
+}
+
+// MsgAddClassCreators is the Msg/AddClassCreators request type.
+// Since Revision 1
+message MsgAddClassCreators {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string authority = 1;
+
+  // creators is the list of addresses to add to the class creator list.
+  repeated string creators = 2;
+}
+
+// MsgAddClassCreatorsResponse is the Msg/AddClassCreators response type.
+// Since Revision 1
+message MsgAddClassCreatorsResponse {}
+
+// MsgRemoveClassCreators is the Msg/RemoveClassCreators request type.
+// Since Revision 1
+message MsgRemoveClassCreators {
+  option (cosmos.msg.v1.signer) = "authority";
+
+  // authority is the address of the governance account.
+  string authority = 1;
+
+  // creators is the list of addresses to remove the class creator list.
+  repeated string creators = 2;
+}
```

Comment:
Same here. The message should only remove a single class creator at a time but it is still possible to include multiple messages that would remove multiple class creators.